mobile_verifier/tests/integrations/last_location.rs

```diff
@@ -192,6 +192,10 @@ async fn heartbeat_does_not_use_last_good_location_when_more_than_12_hours(
         dec!(1.0)
     );
 
+    // NOTE: A new cache will only query backwards up to 12 hours.
```

Comment:
I don't understand this comment, why will a new cache only query backwards up to 12 hours?

Reply:
This comment was a note for me to remember how to ask you the questions I had.